unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -477,5 +487,39 @@ private void OnWearableColorChanged(Color newColor)
             avatarIsDirty = true;
             view.UpdateAvatarPreview(model.ToAvatarModel());
         }
+
+        private void PlayEquipAnimation(string category)
+        {
+            view.PlayPreviewEmote(
+                GetEquipEmoteByCategory(category),
+                DateTimeOffset.UtcNow.ToUnixTimeMilliseconds());
+        }
+
+        private string GetEquipEmoteByCategory(string category)
+        {
+            string equipEmote = category switch
+                                {
+                                    WearableLiterals.Categories.FEET => GetRandomizedName("Outfit_Shoes_v0", 2),
+                                    WearableLiterals.Categories.LOWER_BODY => GetRandomizedName("Outfit_Lower_v0", 3),
+                                    WearableLiterals.Categories.UPPER_BODY => GetRandomizedName("Outfit_Upper_v0", 3),
+                                    "eyewear" => GetRandomizedName("Outfit_Accessories_v0", 3),
+                                    "tiara" => GetRandomizedName("Outfit_Accessories_v0", 3),
+                                    "earring" => GetRandomizedName("Outfit_Accessories_v0", 3),
+                                    "hat" => GetRandomizedName("Outfit_Accessories_v0", 3),
+                                    "top_head" => GetRandomizedName("Outfit_Accessories_v0", 3),
+                                    "helmet" => GetRandomizedName("Outfit_Accessories_v0", 3),
+                                    "mask" => GetRandomizedName("Outfit_Accessories_v0", 3),
+                                    "skin" => GetRandomizedName("Outfit_Upper_v0", 3),
+                                    _ => string.Empty,
```

Comment:
use categories literals (or create them in WearableLiterals)

Reply:
True! I transferred this piece of logic from the previous backpack and I was not aware about these hardcoded strings. Thanks! Done here: https://github.com/decentraland/unity-renderer/pull/5162/commits/625a8f691ce26fe7d6dd81199d89da9a5e11b165